examples-standalone/grid-charts-integration/src/app/charts/stocks-chart.component.ts

```diff
@@ -2,55 +2,61 @@ import { Component, Input } from "@angular/core";
-              {{ getTitle(serie) + ": " + value }}
-            </ng-template>
-          </kendo-chart-series-item-tooltip>
-        </kendo-chart-series-item>
-      </kendo-chart-series>
 
-      <kendo-chart-category-axis>
-        <kendo-chart-category-axis-item [labels]="{ rotation: 'auto' }">
-        </kendo-chart-category-axis-item>
-      </kendo-chart-category-axis>
+    <div class="row">
+        <div class="col-8">
+            <kendo-chart #chart>
+                <kendo-chart-title text="Details per Stock"></kendo-chart-title>
+                <kendo-chart-series>
+                    <kendo-chart-series-item *ngFor="let serie of selectedSeries" [type]="chartConfiguration.seriesType"
+                        [stack]="chartConfiguration.stack" [gap]="2" [spacing]="0.25" [data]="data" [field]="serie" [name]="getTitle(serie)"
+                        [categoryField]="'symbol'">
+                        <kendo-chart-series-item-tooltip>
+                            <ng-template let-value="value">
+                                {{ getTitle(serie) + ": " + value }}
+                            </ng-template>
+                        </kendo-chart-series-item-tooltip>
+                    </kendo-chart-series-item>
+                </kendo-chart-series>
 
-      <kendo-chart-value-axis>
-        <kendo-chart-value-axis-item [labels]="{ rotation: 'auto' }">
-        </kendo-chart-value-axis-item>
-      </kendo-chart-value-axis>
-      <kendo-chart-legend position="right" orientation="vertical">
-      </kendo-chart-legend>
-    </kendo-chart>
+                <kendo-chart-category-axis>
+                    <kendo-chart-category-axis-item [labels]="{ rotation: 'auto' }">
+                    </kendo-chart-category-axis-item>
+                </kendo-chart-category-axis>
 
-    <select-series
-      [data]="series"
-      (valueChange)="selectedSeries = $event"
-    ></select-series>
+                <kendo-chart-value-axis>
+                    <kendo-chart-value-axis-item [labels]="{ rotation: 'auto' }">
+                    </kendo-chart-value-axis-item>
+                </kendo-chart-value-axis>
+                <kendo-chart-legend position="bottom" orientation="horizontal">
+                </kendo-chart-legend>
+            </kendo-chart>
+        </div>
 
-    <select-chart-type
-      [data]="seriesTypes"
-      [chartName]="chartConfiguration.chartName"
-      (valueChange)="onValueChange($event)"
-    ></select-chart-type>
+        <div class="col-4">
+            <select-series [data]="series" (valueChange)="selectedSeries = $event"></select-series>
+
+            <select-chart-type [data]="seriesTypes" [chartName]="chartConfiguration.chartName" (valueChange)="onValueChange($event)"></select-chart-type>
+
+            <button kendoButton [icon]="'image-export'" (click)="exportChart(chart)">
+                Export as Image
+            </button>
+        </div>
+    </div>
+    <div class="row">

```

Comment:
Using ButtonGroups doesn't seem to be a good idea, as we have too many items and the Group becomes too long.